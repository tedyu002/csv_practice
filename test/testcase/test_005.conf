CSV_INPUT=test_005.conf.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
HEADERS=H1 CHAR(4),H2 VARCHAR(32),H3 VARCHAR(12),H4 INTEGER,H5 BOOL,H6 DATETIME,H7 DOUBLE
CSV_FORMULA=Print H3[2] + H3[3]
CSV_FORMULA=Print H2[3] + H1[0] + H7[2]
SORT_HEADERS=H1,H3
SORT_ORDER=ASC
