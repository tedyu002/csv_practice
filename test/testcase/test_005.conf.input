1997,   Ford," E350 ",800000,  TRUE,2017/06/03 12:30:01,12.35
1991,Mercedes-Benz,S320,2800000,TRUE,2013/06/04 23:30:01,12.66
2003,Mercedes-Benz,C220,1800000,TRUE,1998/10/03 08:33:23,528.69
2016,Volvo,S60,1750000,FALSE,2017/06/09 03:49:00,52.63
1991,Audi,A8,1850000,TRUE,2017/05/15 02:18:37,12.5
2003,Mazda,CX-5,1150000,FALSE,2017/05/15 02:18:52,45.96
2017,Skoda,Yeti,950000,FALSE,2017/04/13 02:18:52,12.52
2017,Skoda,Rapid,958000,TRUE,2017/05/15 07:47:34,15.63
2016,Skoda,Citigo,858000,FALSE,2017/05/15 07:48:37,172.8
1991,Audi,Q7,2250000,TRUE,2017/03/15 07:48:37,15.69
