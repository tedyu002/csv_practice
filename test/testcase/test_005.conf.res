"S320 E350 "
"Ford199112.66"
