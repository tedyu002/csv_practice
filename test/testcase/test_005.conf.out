"1991","A8","Audi",1850000,TRUE,2017/05/15 02:18:37,12.50
"1991","Q7","Audi",2250000,TRUE,2017/03/15 07:48:37,15.69
"1991","S320","Mercedes-Benz",2800000,TRUE,2013/06/04 23:30:01,12.66
"1997"," E350 ","Ford",800000,TRUE,2017/06/03 12:30:01,12.35
"2003","C220","Mercedes-Benz",1800000,TRUE,1998/10/03 08:33:23,528.69
"2003","CX-5","Mazda",1150000,FALSE,2017/05/15 02:18:52,45.96
"2016","Citigo","Skoda",858000,FALSE,2017/05/15 07:48:37,172.80
"2016","S60","Volvo",1750000,FALSE,2017/06/09 03:49:00,52.63
"2017","Rapid","Skoda",958000,TRUE,2017/05/15 07:47:34,15.63
"2017","Yeti","Skoda",950000,FALSE,2017/04/13 02:18:52,12.52
